### DISCUSSION TITLE
Swagger documentation autogeneration

### DISCUSSION THREAD

**Discussion by BBerastegui** [reflex-dev/reflex#1840] [Q&A]
Hi all!

I have a pretty simple API built with FastAPI that I was looking to migrate it to Reflex in order to build a nice frontend to it.

One of the things that I liked the most about FastAPI was the automatic generation of Swagger API docs under `/docs`.

Even though I see mentions in the docs that Reflex uses FastAPI as the backend, I can’t seem to find any reference on how to get that Swagger-generation feature for the API if I migrate to Reflex.

tl;dr: I would like to migrate my FastAPI API to have a frontend with Reflex, while maintaining the API itself and the `/docs` Swagger endpoint. 

Thanks in advance!

**Comment by Lendemor** [comment]
Your API will be on the backend, so you should be able to access your `/docs` there. (`/redoc` is also available in the same way) 

In `dev` mode on `localhost`, you'll find it on `localhost:8000/docs` (assuming you are using the default port for backend)

**Comment by BBerastegui** [comment]
Oh dang. Okay, sorry for this then. I couldn’t find it anywhere in the docs.

Thanks!
